editors/code/package.json

```
@@ -591,1065 +429,202 @@
+						"markdownDescription": "Naga version used for validation.",
+						"default": "main",
+						"type": "string",
+						"enum": [
+							"0.14",
+							"0.19",
+							"0.22",
+							"main"
 						],
-						"minimum": 0
+						"enumDescriptions": [
+							"Naga version 14",
+							"Naga version 19",
+							"Naga version 22",
+							"Version of Naga on main (most recent stable version)"
+						]
 					}
 				}
 			},
 			{
-				"title": "completion",
+				"title": "Diagnostics",
 				"properties": {
-					"wgsl-analyzer.completion.postfix.enable": {
-						"markdownDescription": "Whether to show postfix snippets like `if` and `not`.",
+					"wgsl-analyzer.diagnostics.typeErrors": {
+						"markdownDescription": "Controls whether to show type errors.",
 						"default": true,
 						"type": "boolean"
 					}
 				}
 			},
 			{
-				"title": "completion",
-				"properties": {
-					"wgsl-analyzer.completion.snippets.custom": {
-						"markdownDescription": "Custom completion snippets.",
-						"default": {},
-						"type": "object"
-					}
-				}
-			},
-			{
-				"title": "completion",
+				"title": "Inlay Hints",
 				"properties": {
-					"wgsl-analyzer.completion.termSearch.enable": {
-						"markdownDescription": "Whether to enable term search based snippets.",
-						"default": false,
+					"wgsl-analyzer.inlayHints.enabled": {
+						"markdownDescription": "Whether to show inlay hints.",
+						"default": true,
 						"type": "boolean"
 					}
 				}
 			},
 			{
-				"title": "completion",
-				"properties": {
-					"wgsl-analyzer.completion.termSearch.fuel": {
-						"markdownDescription": "Term search fuel in \"units of work\" for autocompletion (Defaults to 1000).",
-						"default": 1000,
-						"type": "integer",
-						"minimum": 0
-					}
-				}
-			},
-			{
-				"title": "diagnostics",
-				"properties": {
-					"wgsl-analyzer.diagnostics.disabled": {
-						"markdownDescription": "List of wgsl-analyzer diagnostics to disable.",
-						"default": [],
-						"type": "array",
-						"items": {
-							"type": "string"
-						},
-						"uniqueItems": true
-					}
-				}
-			},
-			{
-				"title": "diagnostics",
+				"title": "Inlay Hints",
 				"properties": {
-					"wgsl-analyzer.diagnostics.enable": {
-						"markdownDescription": "Whether to show native wgsl-analyzer diagnostics.",
+					"wgsl-analyzer.inlayHints.parameterHints": {
+						"markdownDescription": "Whether to show inlay hints for the names of function parameters.",
 						"default": true,
 						"type": "boolean"
 					}
 				}
 			},
 			{
-				"title": "diagnostics",
+				"title": "Inlay Hints",
 				"properties": {
-					"wgsl-analyzer.diagnostics.experimental.enable": {
-						"markdownDescription": "Whether to show experimental wgsl-analyzer diagnostics that might have more false positives than usual.",
-						"default": false,
+					"wgsl-analyzer.inlayHints.renderColons": {
+						"markdownDescription": "Show colons.",
```

Please "port" descriptions over from r-a. This one was already in w-a and seems to have been trimmed unnecessarily.

```suggestion
						"markdownDescription": "Whether to render leading colons for type hints, and trailing colons for parameter hints.",
```